README.md

```diff
@@ -210,6 +210,7 @@ the specified context and less than the context for the next release.
 
 | jwst tag            | DMS build | SDP_VER  | CRDS_CONTEXT | Released   | Ops Install | Notes                                         |
 |---------------------|-----------|----------|--------------|------------|-------------|-----------------------------------------------|
+| 1.13.0              | B10.1rc1  | 2023.3.1 | 1176         | 2023-12-15 |             | First release candidate for B10.1             |
```

Comment:
should the `SDP_VER` be `TBD` for now?

Reply:
Let's start with it at 2023.4.0 for now.